pkg/meta/annotations.go

```diff
@@ -83,6 +83,12 @@ const (
 	KSafetyAnnotation   = "vertica.com/k-safety"
 	KSafetyDefaultValue = "1"
 
+	// When set to true, the calculation will be based on the number of all nodes
+	// in the cluster;
+	// otherwise, the webhook will calculate the k-safety value
+	// based on the number of primary nodes in the cluster;
+	RelaxKSafetyCheckAnnotation = "vertica.com/relax-k-safety-check"
```

Comment:
Could we rename the annotation so that the default is true? 
```suggestion
	StrictKSafetyCheckAnnotation = "vertica.com/strict-k-safety-check"
```